build.gradle

```diff
@@ -104,6 +105,21 @@ tasks.withType(JavaCompile).configureEach {
     }
 }
 
+// Spotbugs
+spotbugsMain {
+    reportLevel = 'high'
```

Comment:
It means reporting only high priority issue. Should be changed to `medium` (default) or `low` later.